packages/core/src/Editor.ts

```diff
@@ -276,7 +284,19 @@ export class Editor extends EventEmitter<EditorEvents> {
    * Creates a ProseMirror view.
    */
   private createView(): void {
-    const doc = createDocument(this.options.content, this.schema, this.options.parseOptions)
+    let doc: ProseMirrorNode
+
+    try {
+      doc = createDocument(
+        this.options.content,
+        this.schema,
+        this.options.parseOptions,
+        { errorOnInvalidContent: this.options.enableContentCheck },
+      )
+    } catch (e) {
+      this.emit('contentError', { editor: this, error: e as Error })
+      return
```

Comment:
Unsure what to do with the editor when this happens? Should we keep trying to show it?

Reply:
Mhh. I think we should try to show it? Otherwise older documents may lock an editor when old unparseable nodes are loaded? Maybe we could just disable collaborative editing so users don't sync broken documents up and down?